thorvg/examples/thorvg-example/main/display_main.c

```diff
@@ -0,0 +1,151 @@
+/*
+ * SPDX-FileCopyrightText: 2024 Espressif Systems (Shanghai) CO LTD
+ *
+ * SPDX-License-Identifier: CC0-1.0
+ */
+
+#include "freertos/FreeRTOS.h"
+#include "freertos/task.h"
+#include <stdlib.h>
+#include <dirent.h>
+#include "esp_log.h"
+#include "esp_err.h"
+#include "esp_spiffs.h"
+#include "thorvg_capi.h"
+
+static const char *TAG = "example";
+
+static void init_filesystem(void);
+static void capi_shape_example(void);
+static void capi_lottie_example(void);
+
+#define LOTTIE_SIZE_HOR     300
+#define LOTTIE_SIZE_VER     300
+
+void app_main(void)
+{
```

Comment:
should we initialize the LCD panel as well?

Reply:
> should we initialize the LCD panel as well?

OK, I ran it on P4 and S3 yesterday. I didn't put it here, I'll add it.